README.md

```diff
@@ -1,28 +1,32 @@
-## GDX Looker
+## GDX Looker Snowplow Web Block
 
-This Block represents an instance of the Government of British Columbia’s [LookML](https://docs.looker.com/data-modeling/learning-lookml/what-is-lookml) project used for Web Analytics. 
+This Block represents an instance of the Government of British Columbia’s [LookML](https://docs.looker.com/data-modeling/learning-lookml/what-is-lookml), this project is used for Web Analytics. 
 
 ## Features
 
 Looker is a data vision tool that is used with Snowplow Analytics. It presents the data in various formats such as visuals, tables and maps and allows the user to customize the results with an interactive model.
```

Comment:
For features, it should be about web analytics more specifically, rather than generally about Looker.

Reply:
It is a little shorter but I think it does the trick